apps/google-analytics-4/frontend/src/components/main-app/ChartContent/ChartContent.tsx

```diff
@@ -27,7 +28,7 @@ const ChartContent = (props: Props) => {
 
   const renderChartContent = () => {
     if (error) {
-      return <Note body={error?.message || DEFAULT_ERR_MSG} variant="negative" />;
+      return <ErrorDisplay error={error} />;
```

Comment:
I don't have a strong opinion on this, but wanted to ask the question of whether it makes sense to have the chart header and footer still showing when there are errors, or if we want to handle it more like the content type warnings , where errors are displayed at the `AnalyticsApp` level rather than the `ChartContent` level?

Reply:
Yes great question! I have been going back and forth with this, originally just rendered the error message in place of all analytic components like you said, but then it made me a bit nervous to not show the data range select menu in the case that an error is thrown on selection of one date range and not the others - felt in this case the components would shift around a lot of the error message replaced everything, and the user would lose some context that they could piece together. But I am fine with either way definitely 👍

Reply:
That makes sense to me to keep it as you have it in this PR, so that users have the additional context in the header and footer 👍